docs/node/migrations/offline-migrations/network-snapshots.mdx

```diff
@@ -0,0 +1,89 @@
+
+## Create Snapshot
+
+Network snapshots can be generated using the [kwil-admin snapshot](/docs/ref/kwil-admin/snapshot/create) tool. This tool connects directly to the database to download its state and does not require the `kwild` process to be running. 
+
+```bash
+kwil-admin snapshot create --snapdir /path/to/snapdir
+```
+
+By default, the `snapshot create` command connects to the postgres instance runnning at `localhost:5432` with the default user `postgres` with `trust` authentication mode. To connect to a database running on a different host/port/user/password, use the following command:
+
+```bash
+kwil-admin snapshot create --snapdir /path/to/snapdir --dbname <dbname> --host <dbhost> --port <dbport> --user <dbuser> --password <dbpass>
+```
+
+The snapshot tool generates a snapshot file and a genesis file required to start the new network as shown below.
+
+```bash
+# Running the above command will generate the snapshot file and the genesis.json file in the snapshot directory.
+ls /path/to/snapdir
+genesis.json            kwildb-snapshot.sql.gz
+```
+
+## Genesis File
+
+The generated genesis file includes the genesis app hash, derived from the hash of the uncompressed snapshot file, and the genesis validators set to the validators from the previous network. 
+
+:::warning
+The rest of the configuration in the genesis file is set to defaults, so you may need to update it based on your requirements before starting a new network with it.
+:::
+
+
+```json
+{
+  "chain_id": "kwil-chain-XQ7j4ARv",
+  "initial_height": 1,
+  "app_hash": "5IBNnZLoJJCGLEmWTAVE8/RBK0Ap/F/PmWM9HUo+iTQ=",  // app hash derived from the snapshot
+  "allocs": null,
+  "activations": null,
+  "consensus_params": {
+    ...  // set to defaults
+  },
+  "validators": [ // validators from the previous network
+    {
+      "pub_key": "ec37fc333b52b139ebb1176612900a82937a708a61e99c6c657f0ffe18f8f6f0",
+      "power": 1,
+      "name": "validator-0"
+    }
+  ]
+}
+```
+### Verify App Hash
+
+The app hash set in the genesis file is generated based on the hash of the uncompressed snapshot. You can verify the app hash in the generated genesis file as follows
+
+``` bash
+# Uncompress the snapshot file.
+gunzip /path/to/snapdir/kwildb-snapshot.sql.gz
+
+# Generate app hash using sha256sum.
+shasum -a 256 /path/to/snapdir/kwildb-snapshot.sql
+4c2ecfc145c37a45a1fd21cd38ee84db80d852996fc30438a007ba624387c683  /path/to/snapdir/kwildb-snapshot.sql
+
+# app_hash in genesis file is a hexadecimal representation of the sha256sum hash encoded in base64.
+# hex-> binary -> base64
+echo -n "4c2ecfc145c37a45a1fd21cd38ee84db80d852996fc30438a007ba624387c683" | xxd -r -p | base64
```

Comment:
I think the comment here is more accurately `# app_hash in the genesis file is a base64 representation of the sha256 hash value.`.

Also it would be great if `xxd` weren't needed, so we could convert the other direction:

```
jq -r '.app_hash' genesis.json | base64 -d | shasum -a 256
```